roles/webots/keyboardwalk.role

```diff
@@ -15,6 +15,11 @@ nuclear_role(
   # Platform
   input::SensorFilter
   platform::Webots
+  # Vision
+  vision::VisualMesh
+  vision::GreenHorizonDetector
+  vision::GoalDetector
+  vision::BallDetector
```

Comment:
probably don't want to add this here, we could look later at making a new role with all the parts to do the benchmarking and maybe a module that does fixed actions so the benchmarks should be the same each time.